ci/jenkins/jobs/macros.yaml

```diff
@@ -254,28 +254,6 @@
           ./ci/kind/kind-setup.sh destroy "${{FULL_CLUSTER_NAME}}"
           exit $return_code
```

Comment:
You can also remove builder-flexible-ipam-e2e.

Reply:
Please also remove `ipam-e2e-pending` and `flexible-ipam-e2e` in projects-lab.yaml. Additionally, CONTRIBUTING.md should also be updated as the IPAM Jenkins job will no longer be supported.

Reply:
> Please also remove `ipam-e2e-pending` and `flexible-ipam-e2e` in projects-lab.yaml. Additionally, CONTRIBUTING.md should also be updated as the IPAM Jenkins job will no longer be supported after the merge. cc @luolanzone